node/network/gossip-support/src/lib.rs

```diff
@@ -313,10 +330,26 @@ impl State {
 
 		// issue another request for the same session
 		// if at least a third of the authorities were not resolved
-		self.last_failure = if failures >= num / 3 {
-			Some(Instant::now())
+		if failures >= num / 3 {
+			let timestamp = Instant::now();
+			match self.failure_start {
+				None => self.failure_start = Some(timestamp),
+				Some(first) if first.elapsed() >= LOW_CONNECTIVITY_WARN_DELAY => {
+					tracing::warn!(
```

Comment:
what do you think of printing the same with `info` if below delay?

Reply:
I think it would become pretty noisy. We could do it with `debug`, I suppose.